keps/sig-apps/4212-declarative-node-maintenance/kep.yaml

```diff
@@ -0,0 +1,62 @@
+title: Declarative Node Maintenance
+kep-number: 4212
+authors:
+  - "@atiratree"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+  - sig-autoscaling
+  - sig-cli
+  - sig-node
+  - sig-scheduling
+status: provisional
+creation-date: 2023-09-15
+reviewers:
+  - "@adammw"
+  - "@alvaroaleman"
+  - "@aojea"
+  - "@dbenque"
+  - "@dchen1107"
+  - "@fabiand"
+  - "@kwilczynski"
+  - "@mmerkes"
+  - "@pwschuurman"
+  - "@razo7"
+  - "@rptaylor"
+  - "@sftim"
+  - "@soltysh"
+  - "@thockin"
+  - "@wangzhen127"
+approvers:
+  - "@dchen1107"
+  - "@soltysh"
```

Comment:
Please add @fabriziopandini as approver for SIG Cluster Lifecycle

Reply:
done